nuxeo-core/nuxeo-core-event/src/test/resources/test-shutdown-delay.xml

```diff
@@ -0,0 +1,5 @@
+<component name="org.nuxeo.core.event.workmanagerimpl.shutdown.delay">
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <property name="org.nuxeo.ecm.core.work.shutdown.delay">5000</property>
+  </extension>
```

Comment:
IMHO this deserves documentation (sample: https://github.com/nuxeo/nuxeo/blob/release-9.10/nuxeo-services/nuxeo-platform-usermanager-core/src/main/resources/OSGI-INF/usermanager-properties.xml)

Reply:
(discussed live, this is a test conf, will be documented later if still needed)

Reply:
As it's a test contribution, it has been decided it wasn't necessary.